lib/go-atscfg/parentdotconfig.go

```diff
@@ -167,8 +167,9 @@ func MakeParentDotConfig(
 
 // primarySecondary contains the names of the primary and secondary parent cache groups.
 type primarySecondary struct {
```

Comment:
Should we update the comment for this structure to include what nulls is used for